docs/sources/userguide/dockervolumes.md

```diff
@@ -132,6 +134,11 @@ And another:
 
     $ sudo docker run -d --volumes-from dbdata --name db2 training/postgres
 
+It is important to note that if the `postgres` image contained a directory
+called `/dbdata` then mounting the volumes from the `dbdata` container will
+overlay (hide) the `/dbdata` files from the `postgres` image. Meaning, only
```

Comment:
Might not use `overlay` here since it's also a supported fs type, which of course merges things.